src/views/tickets/UrlAnalysis.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Header, Grid } from 'semantic-ui-react';
+
+// My Components
+import SummaryBox from './SummaryBox';
+import Screenshot from './Screenshot';
+import JSViewer from './JSViewer';
+
+const UrlAnalysis = ({ ticket }) => {
```

Comment:
nit: use `Component` instead of a function to maintain consistency with other components in the app.

Reply:
The only issue with Component is that it makes this a class, and UrlAnalysis is really only dealing with the UI. When we refactor the project, I actually think we should refactor classes that don't have any real logic to functions so they are less verbose and it is more obvious whether it deals with logic or UI

Reply:
Hm... definitely picking one or the other sounds good. On the larger react projects I've worked on, we really didn't have any functions returning JSX. Would be interested to see if there's any advantages or disadvantages etc.

Reply:
Yeah, React's actually trying to move away from using classes at all with their introduction of hooks. These function components are easier to reuse and have no side effects, and by writing logic as React hooks rather than huge React classes you can reuse logic too

Reply:
Not saying we should refactor our project using hooks, we're already too far into using classes haha